js/src/helpers/addPreview.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import * as previewComponents from 'headless-block-components';
```

Comment:
What do you think about this convention for block and component names?

| Block name (without namespace)    | React Component name    |
| :------------- | :----------: |
|  `example` | `Example`   | 
|  `example-block` | `ExampleBlock`   | 
| `even-longer-block-name` | `EvenLongerBlockName`  | 

In GCB, block names are usually in a dash format, like `foo-bar`. It doesn't allow snake_case or camelCase.

Of course, this project will also work without GCB, they'll just have to use the React components in their own blocks, and modify the PHP `render_callback` filter.